_data/cryptocurrencies.yml

```
@@ -69,6 +69,14 @@ websites:
     regions:
       - jp
 
+  - name: Bitforex
```

```suggestion
  - name: BitForex
```
It is capitalised like this on their site:

> BitForex | Safest Bitcoin Trading Platform

---

Done ✅